bodhi/server/schemas.py

```diff
@@ -403,6 +403,22 @@ class SaveReleaseSchema(CSRFProtectedSchema, colander.MappingSchema):
     )
 
 
+class UpdateCriticalPackages(colander.MappingSchema):
+    """
+    An API schema for critical path packages.
+
+    This schema is used by bodhi.server.services.releases.add_critpath_packages()
+    and bodhi.server.services.releases.remove_critpath_packages().
+    """
+
+    packages = Packages(
+        colander.Sequence(accept_scalar=True),
+        location='querystring',
+        missing=None,
```

Comment:
It might be possible to use ```missing=list``` here to generate an empty list. I'm not 100% sure you can use a callable there, so try it out and see what happens.